xbmc/playlists/SmartPlayList.cpp

```diff
@@ -290,6 +291,50 @@ bool CSmartPlaylistRule::ValidateMyRating(const std::string &input, void *data)
   return StringValidation::IsPositiveInteger(input, data) && rating <= 10;
 }
 
+static bool StrToInt(const std::string& str, int& num)
```

Comment:
You decided against an anonymous namespace by intention?

Reply:
no, I wasn't aware at the time.